homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
Doesn't this mean that all multilevel switches will be forwarded to the rollershutter component ?